### DISCUSSION TITLE
Will this new repository have a PDF with the project documentation, like we have in the old repository?

### DISCUSSION THREAD

**Discussion by guiamar92** [dotnet/eShop#107] [General]
for studies

**Comment by jamesmontemagno** [comment]
Yes, we are going to be working on new eBooks for this as it progresses forward ;) No ETA as of right now, but that is the plan

**Comment by ParsaMehdipour** [comment]
I believe that this link contains the PDF:
https://learn.microsoft.com/en-us/dotnet/architecture/microservices/
